app/routes/application.ts

```diff
@@ -19,6 +19,7 @@ export default class ApplicationRoute extends Route {
     await all([
       this.configuration.serverConfigFetch(),
       this.configuration.getFrontendConfig(),
+      this.configuration.getDashboardConfig(),
```

Comment:
`getDashboardConfig` should be inside authenticated route as authentication is required for this.